run_aggregate_loanbooks.R

```diff
@@ -410,6 +421,25 @@ company_aggregated_alignment_net_sda %>%
 company_aggregated_alignment_net <- company_aggregated_alignment_net_tms %>%
   dplyr::bind_rows(company_aggregated_alignment_net_sda)
 
+# generate worst case sector splits
+if (apply_sector_split & sector_split_type_select != "worst_case") {
```

Comment:
actually not, because the sector split is already needed earlier in the process, when preparing the matched prioritised loan book. I know this is a somewhat weird logic, but I think short term it is easiest to generate the worst case split based on another sector split that will not remove any sectors. The worst case split keeps only one sector per company, which is why a company level result is needed that has not yet removed any sectors. If we decide to keep this long term, there should be a more elegant way to generate this, I would agree

Reply:
OOof ok I definitely do NOT understand this, but my NIT comment would be to try to make this a bit simpler quickly, because I can imagine someone getting confused/ thinking this is a bug and altering the logic here

Reply:
yea let me think a bit, I might change this before merging

Reply:
valid point